docs/esa/hubble.rst

```diff
@@ -25,14 +25,68 @@ data, all HST data in the EHST are identical to those in MAST.
 Examples
 ========
 
+A complete description of the following functions is also available using the python help function.
+
+.. code-block:: python
+
+  >>> from astroquery.esa.hubble import ESAHubble
+  >>> esahubble = ESAHubble()
+  >>> help(esahubble)
+  Help on ESAHubbleClass in module astroquery.esa.hubble.core object:
+
+  class ESAHubbleClass(astroquery.query.BaseQuery)
```

Comment:
`help` is a standard python command, there is no need to hardwire this in the docs, instead do link to the API doc, rather than doing this copy paste here.